Rakefile

```diff
@@ -65,7 +70,8 @@ namespace :build do
     transform = RUBY_PLATFORM =~ /darwin/ ? "-s " : "--transform s"
     transform << "',^,#{tar_basename}/,'"
 
-    `tar -C ../manageiq #{transform} -X #{exclude_file} -hcvzf #{tarball} .`
+    # Everything from */tmp/* should be excluded, except for tmp/cache/sti_loader.yml
+    `tar -C ../manageiq #{transform} --exclude-tag='cache/sti_loader.yml' -X #{exclude_file} -hcvzf #{tarball} .`
```

Comment:
I'm not sure what exclude-tag does... I think we want to exclude all of tmp but allow tmp/cache/sti_loader.yml if that's possible....